src/content/learn/react-developer-tools.md

```diff
@@ -64,13 +64,13 @@ yarn global add react-devtools
 npm install -g react-devtools
 ```
 
-Next open the developer tools from the terminal.
+তারপর Terminal থেকে ডেভেলপার টুলসটি ওপেন করুনঃ 
 ```bash
 react-devtools
 ```
 
-It should connect to any local React Native app that's running.
+তারপর এটি আপনার যেকোনো চলমান local React Native অ্যাপ এর সাথে কাজ করবে। 
 
-> Try reloading the app if developer tools doesn't connect after a few seconds.
+> যদি ডেভেলপার টুলসটি আপনার অ্যাপ এর সাথে কয়েক সেকেন্ডের মধ্যে কানেক্ট না হলে আপনার অ্যাপ টি Reload করুন। 
```

Comment:
~~যদি~~ ডেভেলপার টুলসটি আপনার অ্যাপ এর সাথে কয়েক সেকেন্ডের মধ্যে কানেক্ট না হলে আপনার অ্যাপ টি Reload করুন।

Reply:
My suggestion would be replacing কানেক্ট with যুক্ত 
Here connect is not a really tech word I think.

Reply:
@nafistiham Bhai, In our daily life we use কানেক্ট word more than যুক্ত word (Almost always). So I think we can more relate with কানেক্ট